src/js/common/schemaform/FormApp.jsx

```diff
@@ -102,6 +124,27 @@ class FormApp extends React.Component {
     return message;
   }
 
+  redirectOrLoad(props) {
+    const firstPagePath = props.routes[props.routes.length - 1].pageList[0].path;
+
+    // If we're logged in and have a saved / pre-filled form, load that
+    if (props.isLoggedIn) {
+      const currentForm = props.formConfig.formId;
+      const isPrefill = props.prefillsAvailable.includes(currentForm);
+      if (props.savedForms.includes(currentForm) || isPrefill) {
```

Comment:
Yeah, I mean, we just go wherever we'll be going when `fetchInProgressForm` finishes, so the last page they were on for saved forms, and the very first "real" page for a pre-fill page.

:man_shrugging: